industry_benchmarks/input_structures/prepared_structures/water_set/hsp90_kung/PREPARATION_DEATILS.md

```diff
@@ -0,0 +1,9 @@
+
+protein:
+ - used maestro protein preparation to cap N-term and C-term (in the original structure termini are neutral and uncapped)
```

Comment:
Could you add the maestro version used here?

Reply:
done